### DISCUSSION TITLE
Reranker jinaai/jina-reranker-v2-base-multilingual hangs / runs forever

### DISCUSSION THREAD

**Discussion by SD-fan** [open-webui/open-webui#3980] [General]
# Bug Report

## Description

**Bug Summary:**
Setting the reranker to the new jinaai/jina-reranker-v2-base-multilingual model works. But once the reranker model is used, it runs forever. OI hangs and can only be revived via a docker restart.

**Steps to Reproduce:**
Set reranker to jinaai/jina-reranker-v2-base-multilingual
Run inference with a document reference ie. #xyz 
OI hangs / CPU has 75% load / nothing happens 

**Expected Behavior:**
jinaai/jina-reranker-v2-base-multilingual should run properly - same as BAAI/bge-reranker-v2-m3.

**Actual Behavior:**
see D

## Environment

- **Open WebUI Version:** 0.3.8 
- **Ollama (if applicable):** version coming with OI docker

- **Operating System:** Win 11 & WSL docker
- **Browser (if applicable):** FF

**Comment by jars101** [comment]
I see same issue, CPU usage very low and it just hangs processing a single txt

**Comment by D1yt** [comment]
I get the following error pop-up when saving the reranker:
```
Something went wrong :/ This modeling file requires the following packages that were not found in your environment: einops. Run `pip install einops`
```

**Comment by morbificagent** [comment]
Yes jina is "running" here forever too. was hoping to get more performance out of my cpu only system with it as BAAI/bge-reranker-v2-m3 is very slow here.

Any idea how to fix this or any tested reranker which is faster than BAAI/bge-reranker-v2-m3 ?

Many thanks
